src/encoded/static/components/regulome_search.js

```diff
@@ -938,24 +939,45 @@ export class RegulomeSearch extends React.Component {
                 if (snp.chrom === chrom && snp.coordinates.gte === +start && snp.coordinates.lt === +end) {
                     hitSnps[snp.rsid] = {};
                     const populationAlleles = {};
+
                     if (snp.ref_allele_freq) {
-                        Object.keys(snp.ref_allele_freq).forEach((allele) => {
-                            Object.keys(snp.ref_allele_freq[allele]).forEach((population) => {
-                                populationAlleles[population] = [`${allele}=${snp.ref_allele_freq[allele][population]}`];
+                        const ref_allele_tag = Object.keys(snp.ref_allele_freq)[0]
+                        if (Object.keys(snp.ref_allele_freq[ref_allele_tag]).length != 0) {
+                            Object.keys(snp.ref_allele_freq).forEach((allele) => {
+                                Object.keys(snp.ref_allele_freq[allele]).forEach((population) => {
+                                    if (!snp.ref_allele_freq[allele][population]) {
+                                        populationAlleles[population] = [`${allele}=N/A`]
+                                    } else {
+                                        populationAlleles[population] = [`${allele}=${snp.ref_allele_freq[allele][population]}`];
+                                    }
+                                    
+                                        
+                                    
+                                });
```

Comment:
Not specifically asking for a change, but I think the next person to modify this code would likely delete these blank lines because you don’t often find blank lines before the ends of code blocks. I think ESLint would mark these as errors, though I thought regulome used ESLint so maybe I’m wrong about that. The same thing would apply to some blank lines at the ends of code blocks below.

Reply:
I didn't use ESlint. Will use it and fix the format.